### PR TITLE
Make banners translatable

### PR DESCRIPTION
**Pull request by mlovic**
References
===================
 - Implements issue: https://github.com/consul/consul/issues/2738
 - PR used as reference: https://github.com/consul/consul/pull/2619
 
Objectives
===================
Allow banners to exist in multiple languages. This applies to 2 of its fields: `title` and `description`.

Visual Changes
===================
selection_114

Notes
===================
As with the original [Milestones PR](https://github.com/consul/consul/pull/2619), data migration is not implemented: 

> At this moment, the migration that creates the milestones translations table is not migrating the existing data. If this is the desired behaviour, it's OK with it. If not, the migration should have the :migrate_data => true option, as seen in the globalize readme:

### CODE REVIEW

File: app/models/banner.rb [hunk 1]
```diff
@@ -3,6 +3,10 @@ class Banner < ActiveRecord::Base
   acts_as_paranoid column: :hidden_at
   include ActsAsParanoidAliases
 
+  translates :title,       touch: true
+  translates :description, touch: true
+  globalize_accessors locales: [:en, :es, :fr, :nl, :val, :pt_br]
```
Comment (aitbw) [inline app/models/banner.rb]: When you backport this to CONSUL, create a new commit removing `:val` language key, as Valencian is not available as an "out-of-the-box" language

Worth noting that `pt_br` was giving @voodoorai2000 some problems on #1580 so be careful with it

Comment (mlovic) [reply app/models/banner.rb]: Okay, thanks for the heads up :)

**Comment by mlovic** [comment]
I just rebased this branch into 2 commits: 1 refactor, 1 feature commit

**Comment by voodoorai2000** [comment]
Fabulous @mlovic 🎉

**Pull request merged**